cl_headers/CMakeLists.txt

```
@@ -1,6 +1,6 @@
 set(CL_HEADERS_LIB cl_headers)
 set(CLANG_COMMAND $<TARGET_FILE:clang> )
-if(LLVM_USE_HOST_TOOLS)
+if(LLVM_USE_HOST_TOOLS AND NOT OPENCL_CLANG_BUILD_EXTERNAL)
```

should this line be
```suggestion
if(LLVM_USE_HOST_TOOLS AND OPENCL_CLANG_BUILD_EXTERNAL)
```
?

---

In this case, clang will be build as aarch64 target, we need use native x86-64 clang build PCH.

---

ok, LGTM, please confirm that PCH built using native clang can be used on arm